src/pages/index.js

```diff
@@ -41,48 +41,48 @@ function IndexPage() {
                 <div className="bg-gray-900 rounded-lg px-3 py-20 md:col-span-1">
                     <div className="text-center mb-5 mx-auto">
                         <h1 className="font-bold text-white text-3xl mt-4">Your masterpiece,<br />one tap away.</h1>
-                        <p className="text-gray-300">Access all the power of Kontax Cam with just a tap away.</p>
+                        <p className="text-gray-300">Access all the power of Kontax Cam with just a tap.</p>
                     </div>
                     <img className="w-10/12 h-auto mx-auto" src={image1} alt="Kontax Cam User interface" />
                 </div>
 
                 <div id="features" className="bg-gray-900 rounded-lg px-10 py-20 md:col-span-2">
                     <div className="text-center">
                         <h1 className="font-bold text-3xl mt-4">Carefully crafted effects</h1>
-                        <p className="text-gray-300">All of these handcrafted effects, plus more in the future!</p>
+                        <p className="text-gray-300">All of these handcrafted effects, plus more coming soon!</p>
                     </div>
 
                     <div className="grid gap-4 grid-cols-1 md:grid-cols-2 mt-20">
                         {
                             [{
                                 icon: colorIcon,
                                 title: `Colour leaks`,
-                                description: `Colour leaks add a gorgeous film colour overlay into your photo. 
+                                description: `Colour leaks add a gorgeous film colour overlay on your photo. 
```

Comment:
You wouldn't put effects inside of a photo, you would put the effects on them. So I made this change.